### DISCUSSION TITLE
Error - No module named 'pyproj.aoi

### DISCUSSION THREAD

**Discussion by tamsima** [insarlab/MintPy#953] [General]
Hello, I can't run MintPy, i get the error "No module named 'pyproj.aoi'". However, I checked and have pyproj package and proj package installed.. Does anyone know what might be wrong? Thanks a lot!

**Comment by yunjunz** [comment]
Please follow the [bug report template](https://github.com/insarlab/MintPy/blob/main/.github/ISSUE_TEMPLATE/bug_report.md) to provide enough information to help the diagnosis.
